salt/modules/zypper.py

```diff
@@ -2033,3 +2123,85 @@ def list_installed_patches():
         salt '*' pkg.list_installed_patches
     '''
     return _get_patches(installed_only=True)
+
+def list_provides(**kwargs):
+    '''
+    List package provides currently installed as a dict.
+    {'<provided_name>': ['<package_name', 'package_name', ...]}
+    '''
+    if 'pkg.list_provides' in __context__:
```

Comment:
Bug, BTW. :wink: You use `list_list_...` instead of `list_...`. This is why this should be a common variable as I mentioned that above.

Reply:
list_list was the bug. Will be changed